### PR TITLE
CCv0 | Ensure kata-deploy scripts from CCv0 are as close to main as possible

### PR DESCRIPTION
**Pull request by fidencio**
This will save us a lot of time on the main -> CCv0 merges.

### CODE REVIEW

File: tools/packaging/kata-deploy/local-build/Makefile [hunk 1]
```diff
@@ -11,15 +11,13 @@ V := 1
 ARCH := $(shell uname -m)
 
 ifeq ($(ARCH), x86_64)
-EXTRA_TARBALL=cc-cloud-hypervisor-tarball \
+EXTRA_TARBALL=\
 	cc-tdx-kernel-tarball \
-	cc-tdx-qemu-tarball \
```
Comment (wainersm) [inline tools/packaging/kata-deploy/local-build/Makefile:16]: ok, the commit message made me confused. We drop the CC target for tdx too but continue to build the same version from CCv0 (i.e. next merges main -> CCv0 we should keep the CCv0 version/source tree). And the other hypervisors we have the main and CCv0 version/source tree equal. Did I get it right?

File: versions.yaml [hunk 2]
```diff
@@ -191,7 +191,7 @@ assets:
     # yamllint disable-line rule:line-length
     description: "Linux kernel with TDX support -- DO NOT TOUCH on main -> CCv0 merges"
     url: "https://github.com/kata-containers/linux/archive/refs/tags"
-    version: "5.19-plus-TDX"
+    version: "5.15-plus-TDX"
```
[the remark this answers is not in the document]
Comment (fidencio) [reply versions.yaml:194]: Exactly.  The CI machine we have allocated for Jenkins is using a 5.15 kernel and we need to keep the compat between host and guest kernel.  On main, though, our GHA machine is running 5.19 and that's the reason of the divergence between main and CCv0.

So, the path taken is to use the very same names / entries in the versions used by `main`, but stick the the CCv0 version.

File: tools/packaging/kata-deploy/local-build/Makefile (hunk 1 above)
Comment (fidencio) [reply tools/packaging/kata-deploy/local-build/Makefile:16]: You got it right.

There are explicit comments in the versions.yaml file about the bits we cannot change.

**Comment by fidencio** [comment]
QEMU TDX passed but didn't report back: http://10.112.240.228:8080/job/kata-containers-CCv0-qemu-TDX-CRI_CONTAINERD_K8S-PR/1061/

**Comment by fidencio** [comment]
CLH TDX passed but didn't report back: http://10.112.240.228:8080/job/kata-containers-CCv0-cloud-hypervisor-TDX-CRI_CONTAINERD_K8S-PR/1010/

**Review by BbolroC — APPROVED** [review]
LGTM, thanks @fidencio !

**Pull request merged**